tools/chouette/initialize-chouette.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+echo "1. Create /opt/chouette/ folder"
+if [ ! -d "/opt/chouette/" ]; then
+
+  sudo mkdir /opt/chouette
+  response=$?
+  if [ $response -eq 0 ]; then
+      echo "OK"
+  else
+      echo "FAIL"
+  fi
+
+else
+  echo "Directory /opt/chouette/ exists."
+fi
+
+echo "2. Copy chouette.zip to /opt/chouette"
+sudo cp mobi.chouette.command-3.4.8.zip /opt/chouette/
+if [ $? -eq 0 ]; then
+    echo "Copy - OK"
+else
+    echo "Copy - FAIL"
+fi
+
+echo "3. Unzip chouette.zip"
+sudo cd /opt/chouette
+sudo unzip /opt/chouette/mobi.chouette.command-3.4.8.zip -d /opt/chouette/
```

Comment:
zip name/path could be given as arg to script to make it universal across versios